ably/rest_client_integration_test.go

```diff
@@ -250,7 +250,6 @@ func TestRest_RSC7_AblyAgent(t *testing.T) {
 		assert.NoError(t, err)
 
 		opts := []ably.ClientOption{
-			ably.WithEnvironment(ablytest.Environment),
 			ably.WithTLS(false),
 			ably.WithUseTokenAuth(true),
 			ably.WithRESTHost(serverURL.Host),
```

Comment:
Should this use `ably.WithEndpoint(serverURL.Host)`?

Reply:
90c9a2bcf57260c53ea42f5d93cbc688c3c493d7